src/main/webapp/app/quiz-statistics/quiz-statistic/quiz-statistic.component.ts

```diff
@@ -339,45 +341,22 @@ export class QuizStatisticComponent implements OnInit, OnDestroy, DataSetProvide
     }
 
     /**
-     * got to the Template with the next QuizStatistic -> the first QuizQuestionStatistic
+     * got to the template with the next QuizStatistic -> the first QuizQuestionStatistic
      * if there is no QuizQuestionStatistic -> go to QuizPointStatistic
      */
     nextStatistic() {
+        // go to quiz-statistic if the position = last position
         if (this.quizExercise.quizQuestions === null || this.quizExercise.quizQuestions.length === 0) {
-            this.router.navigate([
-                '/quiz/:quizExerciseId/quiz-point-statistic',
-                {
-                    quizExerciseId: this.quizExercise.id,
-                },
-            ]);
+            this.router.navigateByUrl('/quiz/' + this.quizExercise.id + '/quiz-point-statistic');
```

Comment:
I think it's better to use a template expression here:

this.router.navigateByUrl(\`/quiz/${this.quizExercise.id}/quiz-point-statistic\`).